soh/src/overlays/actors/ovl_Obj_Comb/z_obj_comb.c

```diff
@@ -171,7 +191,15 @@ void ObjComb_Wait(ObjComb* this, PlayState* play) {
     s32 dmgFlags;
 
     this->unk_1B0 -= 50;
-    if (this->unk_1B0 < 0) {
+    if (
+        IS_RANDO &&
+        Randomizer_GetSettingValue(RSK_SHUFFLE_BEEHIVES) &&
+        !Flags_GetRandomizerInf(this->beehiveIdentity.randomizerInf)
+    ) {
+        if (this->unk_1B0 <= -5000) {
+            this->unk_1B0 = 1500;
```

Comment:
I'm going to probably migrate this to the new system at some point, can you leave a comment explaining what this is intended for?

Reply:
Normally `this->unk_1B0 = 1500;` is run when you hit the hive which makes it wiggle until `unk_1B0` reaches 0.
I repurposed this by letting it count down past 0 (normally it stops at 0) and when it hits -5000 (100 frames as it counts down by 50 every frame) it gets set to 1500 (as if you had just hit the hive), I also added a check to prevent it from wiggling if the value is negative in rando.